packages/app/src/components/popup/asset-list.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import { Box, Flex, Text, BoxProps } from '@blockstack/ui';
+import { TokenAssets } from '@components/popup/token-assets';
+import { CollectibleAssets } from '@components/popup/collectible-assets';
+
+type Tab = 'tokens' | 'collectibles';
+
+export const AssetList: React.FC = () => {
+  const [currentTab, setCurrentTab] = useState<Tab>('tokens');
+
+  const getTabStyles = (tab: Tab): BoxProps => {
+    if (tab === currentTab) {
+      return {
+        borderColor: 'blue',
+        borderWidth: '0 0 2px 0',
+        color: 'blue',
+      };
+    }
+    return {};
+  };
+
+  const TabHeader: React.FC<{ tab: Tab }> = ({ tab }) => {
+    const tabName = new String(tab);
```

Comment:
new String 🤔  Why do you need to do this? Won't the value of tab have a `toString` method.

or when casting you can just pass to `String` without the new?